### ISSUE TITLE
there are nan values

### ISSUE THREAD

**Issue by Winnie202** [open-mmlab/mmsegmentation#1896]
I don't know why there are nan values，train config is upernet_convnext_large_fp16_640x640_160k_ade20k.py
![2022-08-09 19-09-32屏幕截图](https://user-images.githubusercontent.com/86768013/183634430-8f5a536b-91c5-4c6a-bd44-cdb0a6e3ea47.png)

**Comment by xiexinch** [comment]
Hi @Winnie202,
Thanks for your error report and we appreciate it a lot.
Did you make any modifications to the code or config? Is the num_classe of decode_head the same as the number of classes in your dataset? If possible, is it convenient to provide more details? This will help us locate the problem.

**Comment by Winnie202** [comment]
> 
Yes, I modified the category in the configuration file to be consistent with my data, this is my json file
![2022-08-10 10-00-59屏幕截图](https://user-images.githubusercontent.com/86768013/183794120-b6d2cc50-4ea1-4d23-9993-8b06c8fee4cb.png)
![2022-08-10 10-01-16屏幕截图](https://user-images.githubusercontent.com/86768013/183794176-3a6ba863-d76c-4b13-913e-07a3ad734967.png)
![2022-08-10 10-01-37屏幕截图](https://user-images.githubusercontent.com/86768013/183794211-826c56c9-7d2c-475e-934e-140715566e10.png)

**Comment by yxchng** [comment]
Any update on this?
